app/pages/gsoc/gsoc2022/[id].js

```diff
@@ -1,11 +1,15 @@
 import Head from "next/head";
+import Image from "next/image";
 import { fetchAPI } from "../../../lib/api";
 import { contributorList } from "../../../lib/leaderboard";
-import { Container, Col } from "react-bootstrap";
+import { Container, Col, Table } from "react-bootstrap";
 import styles from "../../../styles/Leaderboard.module.css";
```

Comment:
This is a "demonstration page" for the components.

The import should import the components ONLY and then laid them out using simple JSX / bootstrap.

I suggest you include 1 instance of the full size   `leaderboard`   and 2 instances of the `leaderboardcompact` on this page.